source/imaer-gml/src/test/java/nl/overheid/aerius/gml/GMLCalculationSetOptionsReaderTest.java

```diff
@@ -76,7 +78,7 @@ void testReadCalculationSetOptionsNullCalculationOptions() {
     final FeatureCollection featureCollection = mock(FeatureCollection.class);
     final MetaData metaData = mock(MetaData.class);
     final IsCalculationMetaData calculationMetaData = mock(IsCalculationMetaData.class);
-    when(calculationMetaData.getCalculationType()).thenReturn("PERMIT");
+    when(calculationMetaData.getCalculationType()).thenReturn(CALCULATION_METHOD_FORMAL);
```

Comment:
Would have expected this method to mock `getCalculationMethod()` instead with the rename, and perhaps another test method to check that it falls back to `getCalculationType()`.

Reply:
Yes should use new method.